files/galaxy/dynamic_rules/usegalaxy/total_perspective_vortex/destinations.yml

```diff
@@ -3,24 +3,39 @@ destinations:
   interactive_pulsar:
     cores: 24 # arbritrary max
     mem: 128 # arbritrary max
-    #runner: pulsar_embedded
-    #container_monitor_result: callback
-    #docker_auto_rm: True
-    #docker_enabled: True
-    #docker_memory: 4
-    #docker_net: bridge
-    #docker_set_user:
-    #docker_sudo: False
-    #outputs_to_working_directory: False
-    #require_container: True
-    #submit_request_memory: 4
-    submit_requirements: GalaxyDockerHack == True && GalaxyGroup == "interactive"
-    #tmp_dir: True
+    tmp_dir: true
+    docker_enabled: true
+    docker_sudo: false
+    docker_net: bridge
+    docker_auto_rm: true
+    docker_set_user: ""
+    require_container: true
+    outputs_to_working_directory: false
+    container_monitor_result: callback
+    submit_requirements: 'GalaxyDockerHack == True && GalaxyGroup == "interactive"'
     scheduling:
       prefer:
         - docker
         - interactive
 
+  embedded_pulsar_docker:
+    runner: pulsar_embedded
```

Comment:
Maybe it's better to specify the runners in all destinations since we might change to TPV destinations only, soon?

Reply:
Yes, that is also a good idea.